src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -272,6 +272,9 @@ class WorkItemListPage {
   get filterDropdown () {
     return  element(by.id("wi_filter_dropdown"));
   }
+  filterDropdownId () {
```

Comment:
This for just another way to test by different approach , Well when you manually go to Link item everything is clickable , So i have added multiple methods to verify that scenario!